examples/common/src/examples/detection.rs

```diff
@@ -2,11 +2,14 @@
 // Licensed under the MIT license.
 
 use chariott_common::error::{Error, ResultExt};
+use chariott_proto::common::{Blob, InvokeFulfillment, InvokeIntent};
 use prost::Message;
 
-use crate::chariott::{proto::common as common_proto, value::Value};
+use crate::chariott::value::Value;
 
-use super::proto::detection as detection_proto;
+use super::proto::detection::{
+    DetectEntry, DetectRequest as ProtoDetectRequest, DetectResponse as ProtoDetectResponse,
```

Comment:
nit: Consider renaming the aliases from `Proto*` to `*Message` for consistency.

Reply:
Done in 44ce11f2310210263841d7709e36adec95269785